src/app/forms/templates/psv/psv-annual.template.ts

```diff
@@ -222,6 +222,67 @@ export const PsvAnnual: FormNode = {
                 { value: false, label: 'No' }
               ],
               children: []
+            },
+            {
+              name: '',
+              label: 'Emissions',
+              type: FormNodeTypes.SECTION,
+              children: []
```

Comment:
We might want children to be optional or default to [] to make this file a bit cleaner